README.md

```diff
@@ -72,6 +74,7 @@ This event is triggered after a booking has been made within the Healthengine pl
       "dob": "1970-01-01",
       "email": "noreply@healthengine.com.au",
       "firstname": "Jane",
+      "gender": null | "Female" | "Male",
```

Comment:
Our system can support "Other" as an option when enabled.

Reply:
Good catch I totally forgot about that